proposals/Reliable Mutability Primitive.md

```diff
@@ -0,0 +1,166 @@
+# Reliable Mutability Primitive
+
+Authors: [@gozala](https://github.com/gozala)
+
+Initial PR: https://github.com/protocol/web3-dev-team/pull/19/
```

Comment:
Maybe add a note that this is part of the larger story in https://github.com/protocol/web3-dev-team/pull/42 ?

Reply:
I am not sure this would be accurate. I think there is a merit to reliable mutability primitive whether we decide to do human readable names or not.